.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": false,
+  "semi": true,
+  "trailingComma": "es5"
+}
```

Comment:
I have different Prettier default settings, so without a `.prettierrc` file, the formatting is not consistent.